### DISCUSSION TITLE
Idea: Overlay offering suggestions

### DISCUSSION THREAD

**Discussion by FetchFast** [espanso/espanso#1631] [Show and tell]
After doing 
```
$cat ~/Vault/*/*.md > merged.txt
$tr ' ' '\n' < merged.txt | awk 'length>4' | sort | uniq -c | sort -r | head -n 100
```
I now have a list of words to shorten. I pasted these into chatgpt, gave the espanso config file structure and now I've got the top longest words in espanso. 

But I have to actually remember to use them.
This is where an overlay would be helpful to suggest and remind those shorter ways to type words. 

I thought the built in dictionary in OSX would be able to do this, but in testing, it doesn't seem to do what I expected, offering suggestions, but not actually applying them. Perhaps this approach might work better on the platform you use?

**Comment by AucaCoyan** [comment]
Hello!
To understand better, you mean some bubble that looks like this:
![image](https://github.com/espanso/espanso/assets/30557287/0f72ac2a-eb20-4473-b258-526bf37c240e)

If it that's what you want, that is outside the current scope of the project. It's too difficult to implement and highly platform+app dependent
